actors/builtin/miner/miner_actor.go

```diff
@@ -691,16 +696,21 @@ func (a Actor) ConfirmSectorProofsValid(rt Runtime, params *builtin.ConfirmSecto
 
 		newSectorNos := make([]abi.SectorNumber, 0, len(preCommits))
 		for _, precommit := range preCommits {
-			// initial pledge is precommit deposit
-			initialPledge := precommit.PreCommitDeposit
+			// compute initial pledge
+			duration := precommit.Info.Expiration - rt.CurrEpoch()
```

Comment:
Just so it's super-clear, I suggest setting an `ActivationEpoch := rt.CurrEpoch()` var here, using it for the calculation and setting in sector info below.